### ISSUE TITLE
High Availability across DC's & geographic distribution

### ISSUE THREAD

**Issue by yvesgoeleven** [Particular/docs.particular.net#261]
This is probably only for the really big deployments, but a lot can be said (and done) about this type of application, where failover across DC's, or even across clouds is required. Same for systems that need to run in parallel across multiple geographic regions.

**Comment by mauroservienti** [comment]
Is this somewhat related to #368?

**Comment by yvesgoeleven** [comment]
yes, same thing really

**Comment by mauroservienti** [comment]
closing as duplicate of #368
